mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Utils.java

```diff
@@ -16,11 +16,13 @@
 public class Utils {
 
     public static void hideSoftKeyboard(Activity activity) {
-        InputMethodManager inputMethodManager =
-                (InputMethodManager) activity.getSystemService(
-                        Activity.INPUT_METHOD_SERVICE);
-        inputMethodManager.hideSoftInputFromWindow(
-                activity.getCurrentFocus().getWindowToken(), 0);
+        if (activity.getCurrentFocus() != null) {
+            InputMethodManager inputMethodManager =
```

Comment:
Wouldn't it be much better if we use `hideSoftKeyboard(Activity activity)` method from `Utils` class?

Reply:
I am already using that method just modified it to check for null focus

Reply:
Ohh! Yes you are, it is my mistake.